resources/main.qml

```diff
@@ -136,22 +136,21 @@ ApplicationWindow {
                     anchors.fill: parent
 
                     ColumnLayout {
-                        spacing: PhyTheme.marginBig
-                        Layout.margins: PhyTheme.marginBig
+                        spacing: PhyTheme.marginRegular
+                        Layout.margins: PhyTheme.marginRegular
+                        Layout.fillWidth: true
 
                         Label {
                             text: icon
                             color: PhyTheme.teal1
                             font.family: icons.font.family
-                            font.pointSize: Math.max(0.05 * parent.width, 48)
+                            font.pointSize: icons.font.pointSize * 4
                             Layout.alignment: Qt.AlignHCenter
                         }
                         Label {
                             text: name
-                            elide: Text.ElideRight
+                            wrapMode: Text.WordWrap
                             color: PhyTheme.white
-                            font.pointSize: Math.max(0.015 * parent.width, 24)
-                            Layout.alignment: Qt.AlignHCenter
                             Layout.fillWidth: true
```

Comment:
Same here, it looks too small for me:
![20231019_080753 (1)](https://github.com/phytec/qtphy/assets/65712191/6394775e-2e66-4ff5-bff8-75530dbb5e3a)

```suggestion
                            font.pointSize: Math.max(0.015 * parent.width, 24)
                            horizontalAlignment: Text.AlignHCenter
                            Layout.fillWidth: true
```
With suggested changes, it looks like this:
![20231019_085115 (1)](https://github.com/phytec/qtphy/assets/65712191/30c0f1df-18c8-4f55-94b9-b31cd33caa9f)
This is on a 800x480 display, what do you think?

Reply:
I should have mentioned that I plan to use Qt's global scaling mechanisms to adjust the application's font sizes, instead of manually tweaking each label. This has also the benefit of other elements and controls scaling proportionally.

Could you try running qtphy with `QT_SCALE_FACTOR=2` and see if that looks better?

Reply:
I've also changed the cards' name label styling to use `horizontalAlignment: Text.AlignHCenter` as you suggested, and applied HTML header styling to it. That should do the trick without worrying about exact font sizes.

Reply:
Yes this is better now. Does it make sense to make a card name appear bigger than the description?
![20231019_095851](https://github.com/phytec/qtphy/assets/65712191/5af0522c-ab42-46aa-a40a-a08b651a88bf)

Tried bold name, but IMHO it is not really a sufficient change.
![20231019_100636](https://github.com/phytec/qtphy/assets/65712191/e0c69365-be47-4c99-9515-6d3c94653232)

Reply:
Please refetch the latest changes and test again.